ui/components/app/transaction-decoding/components/ui/copy-raw-data/index.scss

```diff
@@ -0,0 +1,27 @@
+.copy-raw-data {
+    display: flex;
+    align-items: center;
+    &__button {
+        border-radius: 20px;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        padding: 8px;
+        margin-top: 5px;
+        background-color: transparent;
+
+        &:hover {
+        background-color: #f2f3f4;
```

Comment:
We use sass variables for colours. See `ui/css/design-system/colors.scss`

Reply:
`#f2f3f4` has a variable (`$Gray-000`), but we notice `#ededed` has no variable and is used elsewhere. What should we do? Make a new variable? Or find an existing variable that is similar?

Reply:
we should find an existing variable which is similar, can be done as part of the PR to address https://github.com/MetaMask/metamask-extension/pull/12881#issuecomment-983474180